handler.py

```diff
@@ -137,17 +150,13 @@ def lambda_handler(event, context):
                 if ev is not None:
                     events.append(ev)
 
-            # send to Axiom
-            result = push_events_to_axiom(events)
-            if result.status != 200:
-                raise f"Unexpected status {result.status}"
-            else:
-                print(f"Ingested {len(events)} events")
+            if len(events) >= 10000:
```

Comment:
what if one file has > 10k lines? shouldn't this be ran in the loop, after the event is appended?